tests/Kernel/CorporateBlocks/CorporateBlocksSiteSwitcherTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\CorporateBlocks;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Test site switcher block rendering.
+ */
+class CorporateBlocksSiteSwitcherTest extends CorporateBlocksTestBase {
+
+  /**
+   * Array containing data to merge with config object.
```

Comment:
We are not "merging" with the config object. We are directly setting this array as the data of the config. And this method can also go to the base class no? Since both children use this.

Please rephrase to something like:

```
Test data for the corporate block.
```

Reply:
Corrected.
Added abstract method in CorporateBlocksTestBase